packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -152,6 +153,27 @@ export class Meter implements types.Meter {
     this._metrics.set(name, metric);
   }
 
+  /**
+   * Provide a pre-computed re-useable LabelSet by
+   * converting the unordered labels into a canonicalized
+   * set of lables with an unique identifier, useful for pre-aggregation.
+   * @param labels user provided unordered Labels.
+   */
+  labels(labels: types.Labels): types.LabelSet {
+    let keys = Object.keys(labels).sort();
+    let identifier = keys.reduce((result, key) => {
+      if (result.length > 2) {
+        result += ',';
+      }
+      return (result += key + ':' + labels[key]);
+    }, '|#');
+    let sortedLabels = {} as types.Labels;
```

Comment:
Can this work as `let sortedLabels: types.Labels = {}`? That's generally a better pattern because it makes sure that the object actually fits the type rather than doing a forceful cast to it.

Reply:
@draffensperger Thanks, will fix it.